riscv/Cargo.toml

```diff
@@ -26,3 +26,4 @@ critical-section-single-hart = ["critical-section/restore-state-bool"]
 [dependencies]
 critical-section = "1.1.2"
 embedded-hal = "1.0.0"
+riscv-pac = { path = "../riscv-pac", version = "0.1.2", default-features = false }
```

Comment:
Do we want to add an extra `feature` to the `riscv` crate to enable the vectors feature in `riscv-pac`?

Reply:
So far, `riscv-pac` only has a default feature to opt out of procedural macros (i.e., `syn`, `proc-macro2`, and `quote` dependencies). However, I don't know if this effort is worthy it, as in the end, you will end up using `riscv-rt`, which depends on procedural macros.

Reply:
maybe we could do something like:
```
[features]
default = ["riscv-pac/default"]
```